plugins/trace_api_plugin/include/eosio/trace_api/trace.hpp

```diff
@@ -61,7 +103,13 @@ namespace eosio { namespace trace_api {
 
 FC_REFLECT(eosio::trace_api::authorization_trace_v0, (account)(permission))
 FC_REFLECT(eosio::trace_api::action_trace_v0, (global_sequence)(receiver)(account)(action)(authorization)(data))
+FC_REFLECT(eosio::trace_api::action_trace_v1, (global_sequence)(receiver)(account)(action)(authorization)(data)(return_value))
 FC_REFLECT(eosio::trace_api::transaction_trace_v0, (id)(actions))
+FC_REFLECT(eosio::trace_api::transaction_trace_v2, (id)(actions))
 FC_REFLECT_DERIVED(eosio::trace_api::transaction_trace_v1, (eosio::trace_api::transaction_trace_v0), (status)(cpu_usage_us)(net_usage_words)(signatures)(trx_header))
+FC_REFLECT_DERIVED(eosio::trace_api::transaction_trace_v3, (eosio::trace_api::transaction_trace_v2), (status)(cpu_usage_us)(net_usage_words)(signatures)(trx_header))
 FC_REFLECT(eosio::trace_api::block_trace_v0, (id)(number)(previous_id)(timestamp)(producer)(transactions))
+FC_REFLECT(eosio::trace_api::block_trace_v2, (id)(number)(previous_id)(timestamp)(producer)(transactions))
 FC_REFLECT_DERIVED(eosio::trace_api::block_trace_v1, (eosio::trace_api::block_trace_v0), (transaction_mroot)(action_mroot)(schedule_version)(transactions_v1))
+FC_REFLECT_DERIVED(eosio::trace_api::block_trace_v3, (eosio::trace_api::block_trace_v2), (transaction_mroot)(action_mroot)(schedule_version)(transactions_v3))
```

Comment:
Why all these types? Why are you skipping numbers?

Reply:
I'm not skipping numbers, block_trace_v2 also used.